fcrepo-auth-webac/src/test/resources/spring-test/security.xml

```diff
@@ -32,9 +30,6 @@
         <ref bean="servletRealm"/>
         <ref bean="webacRealm"/>
       </list>
-    </property>
-      <property name="authenticator.authenticationStrategy">
-      <bean class="org.apache.shiro.authc.pam.FirstSuccessfulStrategy"/>
     </property>
```

Comment:
The closing `</property>` tag should also probably go away.

Reply:
@awoods I think the diff turned the closing tag for `authenticator.authenticationStrategy` into the closing tag for `realms` instead.